docs/learning/WriteAConnector.md

```diff
@@ -287,21 +291,21 @@ If an ExternalSource is not found, insert one using
 _Example:_
 
 ```ts
-[[include:Sychronizer-getOrCreateExternalSource.example-code]]
+[[include:Sychronizer-getOrCreateExternalSource.cf-code]]
```

Comment:
@ben-polinsky , does this seem right? if so feel free to resolve this comment

Reply:
@aruniverse thanks for having a look.  
Please see link from cf pipeline.  https://dev.azure.com/bentleycs/iModelTechnologies/_build/results?buildId=3062111&view=artifacts&pathAsName=false&type=publishedArtifacts  I am generating the snippets there with the 'cf-code' extension.
![cf-code](https://github.com/user-attachments/assets/3fa6f883-e69b-4c3b-bee8-f060bd323d04)
  It is true I made that extension up out of thin air.  I was trying to distinguish the snippets generated from connector-framework builds from the static ones we copied to the "snippets" folder.